lib/SILOptimizer/Differentiation/JVPCloner.cpp

```diff
@@ -779,7 +826,7 @@ class JVPCloner::Implementation final
     auto &diffBuilder = getDifferentialBuilder();
     auto loc = dvi->getLoc();
     auto tanVal = materializeTangent(getTangentValue(dvi->getOperand()), loc);
-    diffBuilder.emitDestroyValue(loc, tanVal);
+    diffBuilder.emitDestroyOperation(loc, tanVal);
```

Comment:
Should we use `emitDestroyValueOperation`? IIRC `emitDestroyOperation` is for both buffers and objects. Here we are only dealing with objects.